telco-core/configuration/reference-crs/required/networking/tuned-queues.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: tuned.openshift.io/v1
+kind: Tuned
+metadata:
+  name: configuration-nic
+  namespace: openshift-cluster-node-tuning-operator
+spec:
+  profile:
+  - data: |
+      [main]
+      summary=Configuration to set to 32 the nb of interrupt per NIC
+      include=openshift-node-performance-$name
+      [net]
+      type=net
+      # Update the NIC name pattern
+      devices_udev_regex=^INTERFACE=ens1f(0|1)
+      channels=combined 32
```

Comment:
During discussion it was floated that 16 may be sufficient. Do we have data to back up either choice?

Reply:
Not really, this is a conservative recommendation. We can start with 16 as that is what we recommend as a starter point for schedulable control plane as well.